serverless/event_sources/knative-event-sources.adoc

```diff
@@ -1,31 +1,30 @@
 include::modules/serverless-document-attributes.adoc[]
 [id="knative-event-sources"]
-= Getting started with event sources
+= Getting started with Event Eources
 include::modules/common-attributes.adoc[]
 :context: knative-event-sources
 toc::[]
 
-An _event source_ is a resource object that acts as a link between an event producer and a _sink_. A sink can be a Knative service, channel or broker that receives events from an event source.
+An Event Source is a resource object that acts as a link between an event producer and a _sink_. A sink can be a Knative Service, Channel, or Broker that receives Events from an Event Source.
```

Comment:
@jboxman would you know if we have rules about capitalization for these terms?
I think Service, Channel, Broker and Event should be. Ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#event-v1-core
I'm not sure if Event Source should maybe be EventSource instead? WDYT?
Looping in @matzew @lberk @devguyio @aliok for the discussion; whatever we decide here we should also drive in upstream because we're inconsistent there quite often.

Reply:
since there's no explicit `Kinda` called `EventSource` I like to think of it as a concept. And first similar thing that came to mind of a concept was [ingress controllers](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/). After reading it, I'm inclined towards using normal lower case words `event source` during description text, and using `Event Source` capitalized when it's part of a specific event source name "e.g. GitHub Event Source" and of course the proper Kind when needed `GitHubEventSource` .

Reply:
I agree with everything except that we should use "GitHub Event Source" - to me using "GitHub event source" just makes more sense for general use if it's a concept, and GitHubEventSource, PingSource etc are used as appropriate when we're referring to the literal thing? So just 2 ways to use, instead of 3.

Reply:
@abrennan89 yes, you are right, I guess I confused capitalizing a heading "Using GitHub Event Source" which is actually capitalizing a text about the concept. So yeah, 2 ways, not 3.